src/Resources/Resources/ChangeLog.md

```diff
@@ -35,6 +35,9 @@
     - Added `ResourceGroupName` parameter which is required when `ResourceGroup` is specified for `ScopeType`. 
     - Added `ManagementGroupId` parameter which is required when `ManagementGroup` is specified for `ScopeType`. 
 
+## Version 1.7.0
+* Fix bug where New-AzRoleAssignment could not be called without parameter Scope.
+
```

Comment:
These lines are duplicated and should be removed

Reply:
Good catch, thanks!